src/components/ScrollSyncRows/ScrollSyncRow/Sections/ScrollableSection/index.jsx

```diff
@@ -76,4 +77,9 @@ class ScrollableSection extends PureComponent {
   }
 }
 
+ScrollableSection.propTypes = {
+  columns: PropTypes.array.isRequired,
+  onScroll: PropTypes.func,
```

Comment:
if onScroll is an optional prop, you should define a default function in ScrollableSection.defaultProps. WDYT?